### ISSUE TITLE
Release to Noetic and ROS2

### ISSUE THREAD

**Issue by JStech** [crigroup/handeye#5]
Thanks for the great software. We've seen a lot of interest in [MoveIt Calibration](https://github.com/ros-planning/moveit_calibration), which is built on handeye.

Could you please make a Noetic release?

**Comment by fsuarez6** [comment]
### ROS Noetic PRs:
* baldor: ros/rosdistro#27125
* criutils: ros/rosdistro#27126
* handeye: ros/rosdistro#27127

**Comment by fsuarez6** [comment]
@JStech 
About ROS2, is it worth releasing for the three distros (Dashing Diademata, Eloquent Elusor and Foxy Fitzroy)?

**Comment by JStech** [comment]
I'd say definitely Foxy and maybe Eloquent (nice-to-have), but Dashing can be skipped.

For some context, we use this package in MoveIt Calibration. I don't think we'll release MoveIt Cal for Eloquent because OpenCV 3.2 (which ships with Ubuntu 18.04) has buggy pose detection of ArUco boards, but otherwise I think 18.04 and Eloquent are still pretty widely used, so you might want to release for them.

**Comment by JStech** [comment]
The release builds for `criutils` and `handeye` [are failing](http://repositories.ros.org/status_page/ros_noetic_default.html?q=suarez). It seems that the appropriate tags in the release repos weren't created--there's no `debian/ros-noetic-*` tag in either [criutils-release](https://github.com/crigroup/criutils-release) or [handeye-release](https://github.com/crigroup/handeye-release). I'm gonna see if I can debug this any further--I'm a total newbie to all this, but happy to learn more.

**Comment by fsuarez6** [comment]
Good catch. For some reason I failed to push the changes to the `*-release` repos.
I've triggered a new release for `criutils` and `handeye`:
* criutils: ros/rosdistro#27765
* handeye: ros/rosdistro#27766

That should do the trick.
